SU-SOE/soe_profile
# SOE basic theme.

# READY FOR REVIEW 

# Summary
- Changes the SOE Profile to install and use the soe_basic theme
- I am not sure where this was dropped in the last build but it slipped through the cracks somewhere.

# Needed By (Date)
- July 9th

# Urgency
- Medium/High

# Steps to Test

1. Check out this branch
2. Do a full site install or run a `drush cim` on an existing install
3. Ensure blocks are all there

# Affected Projects or Products
- SOE

# Associated Issues and/or People
- @jenbreese 
- @imonroe 
- 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
